robot_smach_states/src/robot_smach_states/perception.py

```diff
@@ -14,15 +15,26 @@
 from .util import designators as ds
 
 
-class LookAtEntity(State):
-    def __init__(self, robot, entity, keep_following=False, waittime=0.0):
+class LookAtEntity(smach.State):
```

Comment:
This is a big change. We should check that this does not break anything in e.g. the action server

Reply:
This state is not used right now. So the breaking API change is not a problem.